src/js/gi/config.js

```diff
@@ -6,8 +6,7 @@ module.exports = {
     url: `${environment.API_URL}/v0/gi`,
     settings: {
       headers: {
-        'Content-Type': 'application/json',
-        // 'X-Key-Inflection': 'camel',
+        'X-Key-Inflection': 'camel'
```

Comment:
I was going to add this in a separate PR that would remove any usage of the `camelcase-keys-recursive` package, but wanted to add this here in order to get the newly added `SET_VERION` action to read `version.createdAt` instead of `version.created_at`. Will still get around to the removal separately.